### PR TITLE
Task remove deprecations

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Classes/Controller/ModuleController.php [hunk 1]
```diff
@@ -415,10 +407,11 @@ public function fileReferenceCreateAction($file, $element)
     public function countAction(ServerRequestInterface $request, ResponseInterface $response)
     {
         $requestParameters = $request->getQueryParams();
```
Comment (gilbertsoft) [inline Classes/Controller/ModuleController.php:409]: Not sure if this is problematic conerning SQL injection. Will talk to Olly...

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

File: Classes/Controller/ModuleController.php (hunk 1 above)
Comment (gilbertsoft) [reply Classes/Controller/ModuleController.php:409]: @ohader please have a look at this lines here

Comment (ohader) [reply Classes/Controller/ModuleController.php:409]: * `addslashes` was wrong before - it made things more difficult, but did not protect agains SQL injection
* `TYPO3\CMS\Core\Database\Connection::count($item, $tableName)`
** is using `QueryBuilder` internally
** `$tableName` is passed through `Connection::quoteIdentifier`

Conclusion: remove `addslashes` here

Comment (linawolf) [reply Classes/Controller/ModuleController.php:408]: I removed addslashes and added a comment about this.

File: Classes/Controller/ModuleController.php [hunk 2]
```diff
@@ -406,23 +398,25 @@ public function fileReferenceCreateAction($file, $element)
...
+        $tablename = $requestParameters['table'];
+        /** @var Connection $connection */
+        $connection = GeneralUtility::makeInstance(ConnectionPool::class)
+            ->getConnectionForTable($tablename);
+        $count = $connection->count('uid', $tablename, []);
 
         // Send the response
-        $response->getBody()->write(json_encode($count));
-        return $response;
+        $jsonArray = json_encode($count);
```
Comment (ohader) [inline Classes/Controller/ModuleController.php:418]: `JsonResponse` is encoding internally - this code now would be double-json-encoded...

File: ext_localconf.php [hunk 3]
```diff
@@ -71,7 +71,8 @@
     ]
 );
 // Add TSconfig for new content element wizard
-\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:examples/Configuration/TSconfig/Page/ContentElementWizard.txt">');
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+    '<INCLUDE_TYPOSCRIPT: source="FILE:EXT:examples/Configuration/TSconfig/Page/ContentElementWizard.txt">');
```
Comment (ohader) [inline ext_localconf.php:75]: Maybe use `@include` here...
see https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/TypoScriptSyntax/Syntax/Includes.html

File: Classes/Controller/ModuleController.php (hunk 2 above)
Comment (linawolf) [reply Classes/Controller/ModuleController.php:418]: thanks, I will take it to another PR as this one was already merged

File: ext_localconf.php (hunk 3 above)
Comment (linawolf) [reply ext_localconf.php:75]: thanks, I will take it to another PR as this one was already merged

**Pull request merged**